### PR TITLE
Teachers can remove other teachers and assistants

### PR DESCRIPTION
**Pull request by jsopakar**

### CODE REVIEW

File: app/controllers/assistants_controller.rb [hunk 1]
```diff
@@ -21,6 +21,13 @@ def create
     end
   end
 
+  def destroy
+    authorize! :remove_assistant, @course
+    @assistantship = Assistantship.find(params[:id])
+    @assistantship.destroy!
+    redirect_to organization_course_assistants_path ,notice: 'Assistant removed from course'
```
Comment (mpartel) [inline app/controllers/assistants_controller.rb]: (misplaced comma)

File: app/models/ability.rb [hunk 2]
```diff
@@ -92,11 +92,19 @@ def initialize(user)
...
+      end
+
       cannot :teach, Course
       can :teach, Course do |c|
         return false if c.organization.rejected?
         c.organization.teacher?(user) || c.assistant?(user)
       end
+
+      can :remove_assistant, Course do |c|
+        c.organization.teacher?(user)
```
Comment (mpartel) [inline app/models/ability.rb]: No `!rejected?` and `!acceptance_pending?` checks here?

File: app/views/assistants/_form.html.erb [hunk 3]
```diff
@@ -1,10 +1,9 @@
-<h1>Add a new assistant to <%= @course.title %></h1>
+<h4>Add a new assistant (give username)</h4>
```
Comment (mpartel) [inline app/views/assistants/_form.html.erb]: Why remove the course title?

**Comment by mpartel** [comment]
LGTM other than the above.

File: app/views/assistants/_form.html.erb (hunk 3 above)
Comment (Termanty) [inline app/views/assistants/_form.html.erb]: Visual reason - for me page just looks better without it.

Comment (mpartel) [inline app/views/assistants/_form.html.erb]: Ah, never mind, I was thinking of the form as an entire page but it isn't. Ignore my silliness.

**Pull request merged**